internal/mode/static/nginx/agent/command.go

```diff
@@ -0,0 +1,89 @@
+package agent
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	pb "github.com/nginx/agent/v3/api/grpc/mpi/v1"
+	"google.golang.org/grpc"
+)
+
+// commandService handles the connection and subscription to the agent.
+type commandService struct {
+	pb.CommandServiceServer
+}
+
+func newCommandService() *commandService {
+	return &commandService{}
+}
+
+func (cs *commandService) Register(server *grpc.Server) {
+	pb.RegisterCommandServiceServer(server, cs)
+}
+
+func (cs *commandService) CreateConnection(
+	_ context.Context,
+	req *pb.CreateConnectionRequest,
+) (*pb.CreateConnectionResponse, error) {
+	if req == nil {
+		return nil, errors.New("empty connection request")
+	}
+
+	fmt.Printf("Creating connection for nginx pod: %s\n", req.GetResource().GetContainerInfo().GetHostname())
```

Comment:
Can we log here instead of printing?

Reply:
Yeah, these are meant to be temporary until we fully flesh out these functions. Makes these messages easier to see in the logs.